apps/functions/deadline-submissions/deadline-submission-command/back-office-api-client.js

```diff
@@ -81,12 +97,15 @@ async function getFolderID(caseID, timetableItemName, lineItem) {
  * @returns {Promise<boolean>}
  * */
 async function lineItemExists(caseID, timetableItemName, lineItem) {
-	const response = await got.get(
-		`https://${config.apiHost}/applications/examination-timetable-items/case/${caseID}`
-	);
-	if (!response.ok) {
```

Comment:
Do you need to keep this logic too?

Reply:
I don't think so. I think the unhappy path is handled by the try-catch, so it won't get as far as returning a value because it'll throw.